packages/experiments/src/index.js

```diff
@@ -0,0 +1,64 @@
+const CORE_MODULES_USING_EXPERIMENTS = [
+	'@wordpress/data',
+	'@wordpress/block-editor',
+	'@wordpress/block-library',
+	'@wordpress/blocks',
+	'@wordpress/core-data',
+	'@wordpress/date',
+	'@wordpress/edit-site',
+	'@wordpress/edit-widgets',
+];
+
+const registeredExperiments = {};
+const requiredConsent =
+	'I know using unstable features means my plugin or theme will inevitably break on the next WordPress release.';
+
+export const __dangerousOptInToUnstableAPIsOnlyForCoreModules = (
+	consent,
```

Comment:
Is it possible to ensure calls to this function use a string (as opposed to a variable set to a string). 

```js
// Allowed
__dangerousOptInToUnstableAPIsOnlyForCoreModules(
  'I know using unstable features means my plugin or theme will inevitably break on the next WordPress release.',
  ...
);

// Not allowed
const yeahWhatever = 'I know using unstable features means my plugin or theme will inevitably break on the next WordPress release.';
__dangerousOptInToUnstableAPIsOnlyForCoreModules(
  yeahWhatever,
  ...
);
```

I suspect the answer is no but would rather check :)

Reply:
Unfortunately no :-(

Reply:
Although this is be possible:

```js
import { 
_i_know_using_unstable_features_means_my_plugin_or_theme_will_inevitably_break_on_the_next_WordPress_release,
__dangerousOptInToUnstableAPIsOnlyForCoreModules
} from '@wordpress/experiments';

__dangerousOptInToUnstableAPIsOnlyForCoreModules(
    _i_know_using_unstable_features_means_my_plugin_or_theme_will_inevitably_break_on_the_next_WordPress_release,
   ...
)
```

Reply:
Loving the huge, awkward consent message 😂 

Going the importable variable route would definitely make things easier. I don't know if we _want_ to make things easier? But either way, we'll end up copying and pasting the message in, so the added difficulty is only in locating a place to copy from.